safety/cli.py

```diff
@@ -235,6 +235,21 @@ def inner(ctx, *args, **kwargs):
     return inner
 
 
+def print_deprecation_message():
```

Comment:
Looks good overall! I think it would be useful to add some constants and also to add a docstirng, we are trying to always add docstrings and type hints in the CLI repo when we can.

pseudo code:
```
import click

# Constants
DEPRECATION_DATE = "1 June 2024"
NEW_COMMAND = "`scan`"
LONG_BAR = "+===========================================================================================================================+"
NEWLINE = "\n"

def print_deprecation_message():
    """
    Prints a deprecation warning message informing users that the `check` command will be unsupported beyond a certain date
    and encourages switching to the new `scan` command.
    """
    click.echo(NEWLINE)
    click.echo(click.style(LONG_BAR, fg="yellow", bold=True))
    click.echo(NEWLINE)
    click.echo(
        click.style("DEPRECATED: ", fg="red", bold=True) + 
        click.style(f"this command (`check`) has been DEPRECATED, and will be unsupported beyond {DEPRECATION_DATE}.", fg="yellow", bold=True)
    )
    click.echo(NEWLINE)
    click.echo(
        click.style("We highly encourage switching to the new ", fg="green") + 
        click.style(NEW_COMMAND, fg="green", bold=True) + 
        click.style(" command which is easier to use, more powerful, and can be set up to mimic check if required.", fg="green")
    )
    click.echo(NEWLINE)
    click.echo(click.style(LONG_BAR, fg="yellow", bold=True))
    click.echo(NEWLINE)
```

Reply:
We can probably define the constants within the function here